tests/test_vmapoutput.py

```diff
@@ -0,0 +1,118 @@
+from vmap4fenics import VMAP4Fenics
+import PyVMAP as VMAP
+import numpy as np
+import fenics_concrete
+from os.path import isfile
+import pytest
+
+def test_vmapoutput():
+  # define problem
+  parameters = fenics_concrete.Parameters()
+  parameters['E'] = 30000
+  parameters['nu'] = 0.2
+  parameters['height'] = 300
+  parameters['radius'] = 75
+  parameters['mesh_density'] = 6
+  parameters['log_level'] = 'WARNING'
+  parameters['bc_setting'] = 'fixed'
+  parameters['dim'] = 3
+  experiment = fenics_concrete.ConcreteCylinderExperiment(parameters)
+  problem = fenics_concrete.LinearElasticity(experiment = experiment, parameters = parameters, pv_name = 'test', vmapoutput = True)
+  # define sensors
+  problem.add_sensor(fenics_concrete.sensors.ReactionForceSensorBottom())
+  problem.set_material(name = 'Linear_Concrete',
+            state = 'solid',
+            type = 'concrete',
+            description= 'linear elastic model',
+            material_id = 'linear_concrete_model',
+            idealization = 'continuum',
+            physics = 'solid mechanics'
+            )
+
+  # define displacement
+  displacement_list = [1,5,10]
+  # loop over all measured displacements
+  for displacement in displacement_list:
+    problem.experiment.apply_displ_load(displacement)
+    problem.solve()  # solving this
+
+  metaInfoRead = VMAP.sMetaInformation()
+  unitSystemRead = VMAP.sUnitSystem()
+  testVectorRead = VMAP.VectorTemplateCoordinateSystem()
+  geomPointsRead = VMAP.sPointsBlock()
+  geomElemsRead = VMAP.sElementBlock()
+  elemTypesRead = VMAP.VectorTemplateElementType()
+  intTypeRead = VMAP.VectorTemplateIntegrationType()
+  variableRead = VMAP.VectorTemplateStateVariable()
+
+  # test file exists
+  assert isfile('test/test_VMAP.h5')
```

Comment:
is this the file created by the problem just now?
I am not sure if this path will allways work. Pytest can be called on different levels. It would be good if this path is robust.

Reply:
This should work because the file is created in the same python file as the test and so the relative should be consistent between creation and testing. The only way of making this robust would be, if there was a property in PyVMAP which gives the current path. This is sadly not the case :(